### PR TITLE
DCOS-21932: GPU services

### PR DESCRIPTION
**Pull request by bstavroulakis**
Add GPU field in services table

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [x] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: plugins/services/src/js/utils/ServiceTableUtil.js [hunk 1]
```diff
@@ -64,6 +64,17 @@ function cpusCompareFunction(a, b) {
   return numberCompareFunction(a.getResources().cpus, b.getResources().cpus);
 }
 
+/**
+ * Compare service gpus
+ * @param {Service|ServiceTree} a
```
Comment (DanielMSchmidt) [inline plugins/services/src/js/utils/ServiceTableUtil.js]: Fun fact, we had an eslint rule in one of my last companies for having a certain minimum length of variables (3 it was). This resulted in comparison variables being named `aaa` or `bbb` :trollface:

File: plugins/services/src/js/containers/services/__tests__/ServicesTable-test.js [hunk 2]
```diff
@@ -95,6 +96,15 @@ describe("ServicesTable", function() {
       expect(ReactDOM.findDOMNode(cpusCell).textContent).toEqual("1");
     });
 
+    it("renders resources/stats gpus property", function() {
+      var gpusCell = ReactDOM.render(
```
Comment (DanielMSchmidt) [inline plugins/services/src/js/containers/services/__tests__/ServicesTable-test.js]: It seems like I overlooked this part while trying to move to enzyme, would you mind moving these tests to use enzyme? A no would also be fine, I would file a jira issue then

Comment (weblancaster) [reply plugins/services/src/js/containers/services/__tests__/ServicesTable-test.js]: I would like to see this moved to enzyme too

**Comment by nLight** [comment]
@juliangieseke Jenkins it's not red but PR is red, do you have an idea why this could happen? 🤔

**Comment by juliangieseke** [comment]
Jenkins is reporting unstable builds as fails to github (see my explanation in slack yesterday night)

**Comment by juliangieseke** [comment]
> Nodes Page -- Nodes grid -- Filters nodes grid -- doesnt display any nodes

That one again…

**Comment by nLight** [comment]
I must have missed the failed test glancing over the test results, sorry. Re-triggered

**Comment by juliangieseke** [comment]
In Jenkins UI its Yellow = Unstable = Flaky :)

**Pull request merged**